src/schoology-extractor/schoology_extractor/main.py

```diff
@@ -165,8 +171,14 @@ def main():
         )
 
         if succeeded:
-            # TODO: use correct file path, and use DatFrame instead of list, in FIZZ-103
-            _create_file_from_list(_get_submissions, "submissions.csv")
+            assignments : pd.DataFrame = result_bucket["assignments"]
+
+            for assignment in assignments["SourceSystemIdentifier"].tolist():
+                shared_data["current_assignment_id"] = assignment
+                shared_data["current_section_id"] = section_id
+                assignment_id = int(assignment)
+                submission_file_name = lms.get_submissions_file_path(schoology_output_path, section_id, assignment_id)
+                _create_file_from_dataframe(_get_submissions, submission_file_name)
```

Comment:
It looks like you should be able to use the existing pattern:

```suggestion
                _create_file_from_dataframe(_get_submissions(section_id, assignment_id), submission_file_name)
```

Reply:
You are right... I'll change it. Thanks!